signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlSignerParser.java

```diff
@@ -66,4 +73,15 @@ public List<ArtifactSigner> parse(final String fileContent) {
       throw new SigningMetadataException("Unknown failure", e);
     }
   }
+
+  private static List<SigningMetadata> readSigningMetadata(String fileContent) throws IOException {
```

Comment:
Based on this https://github.com/FasterXML/jackson-dataformats-text/issues/66 this could be simplified to use the readAll method. So something like this should work.

```
final JsonParser parser = YAML_MAPPER.createParser(fileContent);
final List<SigningMetadata> metadataInfoList = YAML_MAPPER.readValues(parser, new TypeReference<SigningMetadata>(){}).readAll();
```

Reply:
👍🏼

Reply:
converted, however, still kept try/resource block.